sdk/identity/Azure.Identity/tests/SharedTokenCacheCredentialTests.cs

```diff
@@ -35,7 +36,7 @@ public async Task VerifyAuthenticationRecordOption()
 
             var mockMsalClient = new MockMsalPublicClient
             {
-                Accounts = new IAccount[] { new MockAccount("nonexpecteduser@mockdomain.com") },
+                Accounts = new List<IAccount> { new MockAccount("nonexpecteduser@mockdomain.com") },
```

Comment:
Why are we refactoring these to `List` instead or array?

Reply:
Reverted